diff -u -N /var/folders/n6/5zsb0nld0z57r18bnv8rnkkr0000gp/T/LIVE-1166533569/v1.Secret.tern-test-apps.app-secret /var/folders/n6/5zsb0nld0z57r18bnv8rnkkr0000gp/T/MERGED-2642781401/v1.Secret.tern-test-apps.app-secret
--- /var/folders/n6/5zsb0nld0z57r18bnv8rnkkr0000gp/T/LIVE-1166533569/v1.Secret.tern-test-apps.app-secret	2026-10-08 12:23:06
+++ /var/folders/n6/5zsb0nld0z57r18bnv8rnkkr0000gp/T/MERGED-2642781401/v1.Secret.tern-test-apps.app-secret	2026-10-08 12:23:06
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  password: '*** (before)'
+  password: '*** (after)'
   username: '***'
 kind: Secret
 metadata:
@@ -8,8 +8,6 @@
     kubectl.kubernetes.io/last-applied-configuration: |
       {"apiVersion":"v1","kind":"Secret","metadata":{"annotations":{},"labels":{"app":"web"},"name":"app-secret","namespace":"tern-test-apps"},"stringData":{"password":"tern-kube-test-not-a-secret","username":"tern-kube-test-user"},"type":"Opaque"}
   creationTimestamp: "2026-10-08T08:10:24Z"
-  labels:
-    app: web
   name: app-secret
   namespace: tern-test-apps
   resourceVersion: "3786"
